modules/cloudfront.js

```diff
@@ -4,6 +4,38 @@ const { assign, compact, concat, flatten, flatMap, get, sortBy, uniq } = require
 
 const { makeWelsh, stripTrailingSlashes } = require('./urls');
 
+/**
+ * Custom cloudfront rules
+ * If any cached url paths need custom cloudfront rules like query strings
+ * or custom cookies to be whitelisted you must define those rules here.
+ */
+const CLOUDFRONT_PATHS = [
+    { path: '*~/link.aspx', isPostable: true, allowAllQueryStrings: true },
+    { path: '/api/*', isPostable: true, allowAllQueryStrings: true },
+    { path: '/funding/funding-finder', isPostable: true, allowAllQueryStrings: true, isBilingual: true },
+    { path: '/funding/programmes', queryStrings: ['location', 'amount', 'min', 'max'], isBilingual: true },
+    { path: '/funding/search-past-grants-alpha', isPostable: true, allowAllQueryStrings: true, isBilingual: true },
+    { path: '/search', allowAllQueryStrings: true, isBilingual: true },
+    { path: '/user/*', isPostable: true, queryStrings: ['token'] }
```

Comment:
For the user endpoint, there are also these query parameters: `s` and `redirectUrl`.

Reply:
These haven't changed so TBH we might not even need it in here as those pages are all uncached but will add it for completeness.